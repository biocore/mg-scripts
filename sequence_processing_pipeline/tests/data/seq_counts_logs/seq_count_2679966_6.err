This is an example .err file produced by seq_counts.sbatch.
Additional details removed.
+ seqtk size REMOVED/working_dir/8edbdee2-da52-4278-af40-267185bbcd7e/TRIntegrateJob/integrated/Test_8_22_2014_R2_example_S2_L007_R2_001.fastq.gz
