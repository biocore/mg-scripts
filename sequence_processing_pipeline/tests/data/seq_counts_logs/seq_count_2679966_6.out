REMOVED/8edbdee2-da52-4278-af40-267185bbcd7e/TRIntegrateJob/integrated/LS_8_22_2014_R2_SRE_S2_L007_R2_001.fastq.gz
64464162	8370238082
